SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -77,8 +77,20 @@ private void SaveValue(string key, string value)
                 {
                     element.Value = value;
                 }
+                SaveConfiguration();
+            }
+        }
+
+        private void SaveConfiguration()
+        {
+            try
+            {
                 configuration.Save();
             }
+            catch (ConfigurationErrorsException e)
+            {
+                AppCenterLog.Warn(AppCenterLog.LogTag, $"Configuration file can't be saved. Failure reason: {e.Message}");
```

Comment:
Looks good but make sure to test SQLite interactions from different processes before we merge.
We might need to spam more logs to see issues (try a for loop).

Reply:
Done. I have created 'for' loop from 0 to 1000000. In loop 'Analytics.TrackEvent' method was called. It was done with two active app instances. No exceptions.

Reply:
Update, SQLite Busy error was reproduced. 28000-30000 logs were tracked as usual, after that the error could appear.

Reply:
What happens after the error? Is the SDK printing it? Is the app crashing? Can you track more events correctly after the errors or is not recoverable?